domain/goal/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/goal/service/GoalService.kt

```diff
@@ -32,16 +32,23 @@ class GoalService(
    * Creates a new [Goal] for the prisoner identified by their prison number, with the data in the specified [CreateGoalDto]
    */
   fun createGoal(prisonNumber: String, createGoalDto: CreateGoalDto): Goal {
```

Comment:
I've left the original `createGoal` (singular) method here on the service class; but currently nothing (except the unit test) uses it. It feels like a convenient method should we ever need to create a single goal , but happy to remove it if we think that would be better?

Everywhere else I've removed the original `createGoal` (singular) type methods, and replaced with `createGoals`